### ISSUE TITLE
Option to skip fetching packages when opening "chocolatey" tab with list of all packages

### ISSUE THREAD

**Issue by MurzNN** [chocolatey/ChocolateyGUI#939]
## Is Your Feature Request Related To A Problem? Please describe.

I open "chocolatey" tab with list of all packages most of the time to search packages by keyword, not via manually browsing most popular packages in the list of all packages. But each time when I open that tab to start search, I need to waste time with seeing initial "Fetching packages..." popup window that fetches first page of most popular packages.

## Describe The Solution. Why is it needed?

Will be good to add an option (maybe even enabled by default) to disable this behavior with fetching most popular packages each time when we open "chocolatey" tab to find desired packages. This should be not only optimize the users' time, but also decrease the amount of unnecessary queries to chocolatey servers.

As replacement of previous most popular packages list we can put the button "Fetch most popular packages" instead of showing the empty list there.

## Additional Context.

N/A

## Related Issues

N/A

**Comment by gep13** [comment]
@MurzNN I believe the feature that you are looking for is here:

https://docs.chocolatey.org/en-us/chocolatey-gui/setup/configuration/features/prevent-preload

If this solves your request, we can get this closed out.

**Comment by MurzNN** [comment]
Yeah, that is exactly what I've desired! Name of this option is not such clear, so I've missed it out.
What are you thoughts about enabling this option by default (with adding "Fetch most popular packages" button instead of list) to decrease amount of unnecessary queries to Chocolatey servers? I think it should show significant decrease of load on server side.
